modules/express/test/unit/clientRoutes/createAddress.ts

```diff
@@ -0,0 +1,64 @@
+import * as sinon from 'sinon';
+
+import 'should-http';
+import 'should-sinon';
+import '../../lib/asserts';
+
+import * as express from 'express';
+
+import { handleV2CreateAddress } from '../../../src/clientRoutes';
+
+import { BitGo } from 'bitgo';
+
+describe('Create Address', () => {
+  function createAddressMocks(res) {
+    const createAddressStub = sinon.stub().returns(res);
+    const walletStub = { createAddress: createAddressStub };
+    const coinStub = {
+      wallets: () => ({ get: () => Promise.resolve(walletStub) }),
+    };
+    return {
+      bitgoStub: sinon.createStubInstance(BitGo as any, { coin: coinStub }),
+      createAddressStub,
+    };
+  }
+
+  it('should return the address object', async () => {
+    const res = { address: 'addressdata' };
+    const { bitgoStub } = createAddressMocks(res);
+    const req = {
+      bitgo: bitgoStub,
+      params: {
+        coin: 'tbtc',
+        id: '23423423423423',
+      },
+      query: {},
+      body: {
+        chain: 0
+      },
+    } as unknown as express.Request;
+
+    await handleV2CreateAddress(req)
+      .should.be.resolvedWith(res);
+  });
+
+  it('should return the address object', async () => {
```

Comment:
nit: seems like this is the same test, but for eth. Do we need both? If so, could you add that to the test name so we know these are actually doing different things?

Reply:
I have removed one test as there was no other difference apart from the coin and in both cases, the behaviour is going to be the same only